tests/simple/simple_test.go

```diff
@@ -119,6 +136,45 @@ func TestSimpleFile(t *testing.T) {
 	if rc == nil {
 		return
 	}
+	var skipTests []string
+	firstFlag := true
+	for _, flagVal := range skipFlags {
+		fileInd := strings.Index(flagVal, "/")
+		splitFile = strings.SplitN(flagVal, "/", 2)
+		fileName := splitFile[0]
+		sectionString := splitFile[1]
+		if fileInd > 0 || sectionString == "" {
+			log.Fatal("skip_test argument must contain at least <file>/<section>, received ", flagVal)
+		}
+		for _, sectionVal := range strings.Split(sectionString, ";") {
+			sections := strings.Count(sectionVal, "/")
+			if sections == 0 {
+				if sectionVal != "" {
```

Comment:
General pattern:  try to handle errors in if-bodies to let the main flow proceed outdented without an "else".  In this case, rewrite as:

```
if sectionVal == "" {
        log.Fatal("...")
}
skipTests = append(...)
```